packages/core/src/index.ts

```diff
@@ -1107,15 +1107,15 @@ export class Hopr extends EventEmitter {
     }
 
     return {
-      pending,
-      losing,
-      winProportion: acked_tickets.len() / (acked_tickets.len() + losing) || 0,
-      unredeemed: acked_tickets.len(),
+      pending: pending.toString(),
+      losing: losing.toString(),
+      winProportion: (acked_tickets.len() / (acked_tickets.len() + losing) || 0).toString(),
+      unredeemed: acked_tickets.len().toString(),
```

Comment:
Would it not be better the other way around? Those are definitely number values, why should those be strings?

Reply:
Makes sense! changed

Reply:
@esterlus @jeandemeusy @mjadach-iv Please approve this change to ensure you are fine with adapting the response data this way.

Reply:
I've removed it. No need to approve anymore